device/esp_tinyusb/test_apps/vendor/main/test_vendor.c

```diff
@@ -21,7 +21,11 @@
 static const char *TAG = "vendor_test";
 
 char buffer_in[64];
+#if (TUSB_VERSION_MINOR >= 17)
```

Comment:
Are the this changes even needed? Updating build dir and work dir seems to fix the problem.

Reply:
Yes, they remove another problem and unlock CI for esp-usb.